src/main/scala/no/ndla/searchapi/service/search/SearchConverterService.scala

```diff
@@ -778,7 +787,24 @@ trait SearchConverterService {
       }
     }
 
-    private def getTaxonomyResourceAndTopicsForId(id: Long, bundle: Bundle, taxonomyType: String) = {
+    private[service] def getGrepContexts(grepCodes: Seq[String], bundle: GrepBundle): List[SearchableGrepContext] = {
+      val grepContext = bundle.kjerneelementer ++ bundle.kompetansemaal ++ bundle.tverrfagligeTemaer
+
+      grepCodes
+        .map(
+          grepCode =>
+            SearchableGrepContext(
+              grepCode,
+              grepContext
+                .find(grepElement => grepElement.kode == grepCode) match {
+                case Some(element) => element.tittel.find(title => title.spraak == "default").map(title => title.verdi)
```

Comment:
Mulig vi skal indeksere fleire språkvarianter av disse titlene. Då må vi ha en mapping mellom språkkoder i grep til våre koder.

Reply:
Enig, men da må vi implementere språk-håndteringa som GDL bruker, for vi bruker "nb", "nn" mens grep bruker "nob", "nno"

Reply:
Det finnes en issue på det. https://github.com/NDLANO/Issues/issues/1637

Reply:
Tenker at språkhåndtering kan kanskje være sin egen issue, for vi bruker også bare "default" språk i ed, så når vi først skal implementere GDL sin løsning, burde vi da implementere den alle steder.